src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -147,20 +147,17 @@ internal static SecurityZone GetFileSecurityZone(string filePath)
         {
             Diagnostics.Assert(Path.IsPathRooted(filePath), "Caller makes sure the path is rooted.");
             Diagnostics.Assert(Utils.NativeFileExists(filePath), "Caller makes sure the file exists.");
-#if CORECLR
+#if !UNIX
             string sysRoot = System.Environment.GetEnvironmentVariable("SystemRoot");
             string urlmonPath = Path.Combine(sysRoot, @"System32\urlmon.dll");
             if (Utils.NativeFileExists(urlmonPath))
             {
                 return MapSecurityZoneWithUrlmon(filePath);
             }
-            return MapSecurityZoneWithoutUrlmon(filePath);
-#else
-            return MapSecurityZoneWithUrlmon(filePath);
 #endif
+            return MapSecurityZoneWithoutUrlmon(filePath);
```

Comment:
it appears you're still hitting alternate data streams on unix.
ClrFacade:  MapSecurityZoneWithoutUrlmon -> ReadFromZoneIdentifierDataStream